src/overlays/actors/ovl_Bg_Goron_Oyu/z_bg_goron_oyu.c

```diff
@@ -8,14 +8,18 @@
 
 #define FLAGS 0x00000030
 
-#define THIS ((BgGoronOyu*)thisx)
-
```

Comment:
We haven't decided what we are going to do with the `THIS`macro, so I suggest to bring it back and use it where appropriate